dandiapi/api/views/serializers.py

```diff
@@ -162,12 +166,25 @@ class Meta:
             'created',
             'modified',
             'dandiset',
+            'release_notes',
         ]
         read_only_fields = ['created']
 
     dandiset = DandisetSerializer()
+    release_notes = serializers.SerializerMethodField()
     # name = serializers.SlugRelatedField(read_only=True, slug_field='name')
 
+    def get_release_notes(self, version: Version) -> str | None:
+        """Extract release notes from version metadata."""
+        return version.metadata.get('releaseNotes')
+
+    def to_representation(self, instance):
+        """Remove release_notes from representation if it's None."""
+        representation = super().to_representation(instance)
+        if representation.get('release_notes') is None:
+            representation.pop('release_notes', None)
+        return representation
+
```

Comment:
We should be able to access the `releaseNotes` directly from the metadata in the client, which would remove the need for this extra logic in this serializer. First, we'll need https://github.com/dandi/dandi-schema/pull/185/ to be released.